webextensions/manifest/background.json

```diff
@@ -61,17 +61,15 @@
               },
               "firefox": {
                 "version_added": "48",
-                "partial_implementation": true,
                 "notes": [
-                  "Only persistent pages are supported.",
+                  "Before version 101, only persistent pages are supported.",
```

Comment:
persistent pages are not yet supported.  they are pref'd off and available to developers to pref on for testing.  we might want to reword that, and have tickets to return to these items later.  maybe something like:

"Only persistent pages are supported.  As of 101 addon developers may turn on event pages for testing purposes"

Reply:
Using the MV3 preference I assume? e.g. "Only persistent pages are supported. As of version 101, extension developers can turn on event pages for testing purposes using the `extensions.manifestV3.enabled` preference."

Reply:
They can also turn on event pages for MV2 using `extensions.eventPages.enabled` and that will be in the blog posts.